PocketKit/Sources/Sync/PersistentContainer.swift

```diff
@@ -36,21 +36,7 @@ public class PersistentContainer: NSPersistentContainer {
         let model = NSManagedObjectModel(contentsOf: url)!
         super.init(name: "PocketModel", managedObjectModel: model)
 
-        switch storage {
-        case .inMemory:
-            storeDescription = NSPersistentStoreDescription(url: URL(fileURLWithPath: "/dev/null"))
-        case .shared:
-            let sharedContainerURL = FileManager.default
-                .containerURL(forSecurityApplicationGroupIdentifier: groupID)!
-                .appendingPathComponent("PocketModel.sqlite")
-
-            Log.debug("Store URL: \(sharedContainerURL)")
-            storeDescription = NSPersistentStoreDescription(url: sharedContainerURL)
-        }
-
-        guard let storeDescription else {
-            fatalError("no store description")
-        }
```

Comment:
Not sure if I'm getting this right, but with this change, it seems that there is a path where, if we initialize with the `shared` storage and, for some reason, the on disk db is not available, we' re still initializing the app with a volatile storage. Is this what we want? cc @bassrock

Reply:
The scenario I coded for was if we ask for .shared storage with a bad groupID, we fall back to .inMemory. Maybe we need a more explicit `if groupID is the default garbage value { fall back to .inMemory }` but that seems specific and brittle.

Reply:
moving a bit of conversation to slack on this